x/auth/types/errors.go

```diff
@@ -5,6 +5,5 @@ import (
 )
 
 var (
-	ErrorInvalidSigner        = sdkerrors.Register(ModuleName, 2, "tx intended signer does not match the given signer")
-	ErrorInvalidGasAdjustment = sdkerrors.Register(ModuleName, 3, "invalid gas adjustment")
```

Comment:
Where should this stuff get moved to @alexanderbez? Maybe just to `types/`?

Reply:
`types/errors` 👍 

That's where we already have core errors defined.